src/Webfactory/Slimdump/Config/FakerReplacer.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Webfactory\Slimdump\Config;
+
+use Faker;
+use Webfactory\Slimdump\Exception\InvalidReplacementOptionException;
+
+/**
+ * This class handles column configurations which have a "replacement" option
+ * @see https://github.com/webfactory/slimdump#configuration for configuration tutorial
+ *
+ * @package Webfactory\Slimdump\Config
+ * @author franziskahahn
+ */
+class FakerReplacer
+{
+    const PREFIX = 'FAKER_';
+
+    /** @var \Faker\Generator */
+    private $faker;
+
+    /**
+     * configuration for method which are not part of faker itself
+     * @var array
+     */
+    private $replacementOptions = [
+        'name' => 'generateFullNameReplacement',
+    ];
+
+    /**
+     * FakerReplacer constructor.
+     * TODO: Add functionality which makes locale configurable
+     */
+    public function __construct()
+    {
+        $this->faker = Faker\Factory::create();
+    }
+
+    /**
+     * check if replacement option is a faker option
+     * static to prevent unnecessary instantiation for non-faker columns
+     * @param string $replacement
+     * @return bool
+     */
+    public static function isFakerColumn($replacement)
+    {
+        if (strpos($replacement, self::PREFIX) === 0) {
```

Comment:
My impulse was to shorten it to `return strpos($replacement, self::PREFIX) === 0;`, but that seems to be a matter of personal taste, so please go along.